package tutorial.orcas.combat.server.require;

enum RequireType {
  REQUIRE_SCENE_BUILD = 101;
  REQUIRE_SCENE_MOVE = 102;
  REQUIRE_SCENE_FETCH = 103;

  REQUIRE_FORMULA_ATTACK = 201;
}
